aptos-move/framework/aptos-framework/sources/signature.move

```diff
@@ -38,8 +37,25 @@ module aptos_framework::signature {
         signature: vector<u8>
     ): (vector<u8>, bool);
 
+    /// Return true if the BLS `signature` on `message` verifiers against the BLS public key `public_key`.
+    /// Returns `false` if:
+    /// - `signature` is not 96 bytes
+    /// - `public_key` is not 48 bytes
+    /// - `signature` or `public_key` are not valid: i.e., (1) they are the identity point, or (2) they are not valid
+    ///    points on the BLS12-381 elliptic curve or (3) they are not prime-order points.
+    /// - `signature` and `public key` are valid but the signature on `message` is not valid.
```

Comment:
feels like we should probably add "valid" functions, abort codes, or both.

Reply:
well we have a validate public key

Reply:
Ah, I don't know how to do this. Any examples I can look at?

Reply:
The Ed25519 Move functions also "do not abort", so I emulated that.